library/core/src/slice/mod.rs

```diff
@@ -2056,6 +2056,56 @@ impl<T> [T] {
         sort::quicksort(self, |a, b| f(a).lt(&f(b)));
     }
 
+    /// Reorder the slice such that the element at `index` is at its final sorted position.
+    ///
+    /// This function has been obsoleted by [`select_nth_unstable`], and will be removed in
+    /// a later release.
+    #[unstable(feature = "slice_partition_at_index", issue = "55300")]
```

Comment:
Could you mark these as deprecated with a suggestion to use `select_nth_unstable` instead?